internal/ingress/controller/kong.go

```diff
@@ -269,8 +274,9 @@ func (n *NGINXController) syncServices(ingressCfg *ingress.Configuration) (bool,
 							s.WriteTimeout = kongIngress.Proxy.WriteTimeout
 						}
 
-						if kongIngress.Proxy.Retries > 0 {
+						if kongIngress.Proxy.Retries > 0 && s.Retries != kongIngress.Proxy.Retries {
```

Comment:
If a kongIngress object sets up ReadTimeout (or any timeout for that matter) as 100000, then this logic will not update the corresponding service object in Kong, right?

Reply:
Nice catch! I've added the comparison for `ConnectTime` as well.